tcms/core/contrib/auth/tests.py

```diff
@@ -79,7 +79,7 @@ def setUpTestData(cls):
                                               password='password')
         cls.logout_url = reverse('tcms-logout')
 
-    def test_logout_then_redirect_to_next(self):
+    def test_logout_then_redirect_to_next(self):  # pylint: disable=invalid-name
```

Comment:
You can safely move the invalid-name disable to the top of this module. It's acceptable to do so for the tests.